simpledotorg/simple-server
# Script to export patient data to DHIS2

**Story card:** [sc-10909](https://app.shortcut.com/simpledotorg/story/10909/write-a-script-to-create-5-10-fake-patient-records-on-dhis2)

## Because

We want to export 3 million patients to DHIS2 sandbox eventually. This PR adds an exporter script to set this up.

## This addresses

This PR adds:
- an exporter for tracked entity data, ie patients
- config files to store local and sandbox ids for DHIS2 elements
- a rake task that exports patient data for all facilities defined in the config as mapped to organisation units on DHIS2

## Test instructions

Run the rake task and check that the corresponding organisation units on DHIS2 have the patients imported correctly.
